app/app.go

```diff
@@ -20,12 +20,13 @@ import (
 	"github.com/ava-labs/avalanchego/utils/ulimit"
 )
 
-const Header = `     _____               .__                       .__
-    /  _  \___  _______  |  | _____    ____   ____ |  |__   ____    ,_ o
-   /  /_\  \  \/ /\__  \ |  | \__  \  /    \_/ ___\|  |  \_/ __ \   / //\,
-  /    |    \   /  / __ \|  |__/ __ \|   |  \  \___|   Y  \  ___/    \>> |
-  \____|__  /\_/  (____  /____(____  /___|  /\___  >___|  /\___  >    \\
-          \/           \/          \/     \/     \/     \/     \/`
+const Header = `
```

Comment:
The skier must stay - for significant UX reasons.

Reply:
dnlgtm